addons/base_import/tests/test_cases.py

```diff
@@ -23,6 +25,13 @@ def sorted_fields(fields):
     recursed = [dict(field, fields=sorted_fields(field['fields'])) for field in fields]
     return sorted(recursed, key=lambda field: field['id'])
 
+def test_module_not_installed(module_name):
```

Comment:
the function is used to skip the test, when 'XLRD' module is not installed and throws warning instead of error. it is used on L262 & L293